components/playlist/browser/resources/components/header.tsx

```diff
@@ -60,8 +62,68 @@ const StyledPlaylistInfo = styled(PlaylistInfo)`
   flex-grow: 1;
 `
 
+const LeoButtonContainer = styled.button`
+  display: contents;
+`
+
+function NewPlaylistButton () {
+  return (
+    <LeoButtonContainer
```

Comment:
Out of curiosity, can you use the built in Leo button components for this, or is there something about them which doesn't work here?

I think the `plain-faint` style might look similar, but I'm not 100% sure :laughing:

Reply:
Oh, that's cool. I always forget about it. Thanks. 

It looks like the button has fixed size based on the ButtonSize props, which makes it bigger than the icon size. Is there anyway to make it fit to the icon?
<img width="339" alt="image" src="https://github.com/brave/brave-core/assets/5474642/251da1d4-8aa4-4ff7-9b5d-f041a828b808">

Reply:
Ah yeah, there's a bit of built in padding, so maybe it won't work here :cry: